src/ResourceManager/DataFactories/Commands.DataFactoryV2.Test/ScenarioTests/RunTests.cs

```diff
@@ -31,5 +31,11 @@ public void TestRunV2()
             RunPowerShellTest("Test-Run");
         }
 
+        [Fact]
```

Comment:
Can you add a test for a successful cancellation?

Reply:
Modified the run test to include it